src/utils/format.ts

```diff
@@ -85,9 +85,11 @@ export function parseAmount(amountFmt: string, amountPrecision = DEFAULT_PRECISI
 
   const groups = /^(\d+)(?:\.(\d+))?$/.exec(amountFmt)
   if (groups) {
-    const [, integerPart, decimalPart = ''] = groups
+    const integerPart = groups[1]
+    let decimalPart = groups[2] || ''
     if (decimalPart.length > amountPrecision) {
-      return null
+      // truncate whatever is over precision
+      decimalPart = decimalPart.slice(0, amountPrecision)
```

Comment:
So we want to change the behaviour for parse amount too, is that right?

Reply:
Even though I'm not depending on this, we had agreed to change. I don't mind removing the change though, just think it makes more sense to truncate rather than return `null`.

What do you say?

Reply:
`null` signals, I can't parse. However, the truncation is a fair approach. And if we agreed, I don't want to change that :)